cypress/integration/index.spec.js

```diff
@@ -1,16 +1,37 @@
 describe('Index page', () => {
   beforeEach(() => {
-    cy.stubRequest('productList.json', 'GET_PRODUCTS_QUERY')
+    cy.route2('POST', Cypress.env('NEXT_PUBLIC_GRAPHQL_URI'), (req) => {
```

Comment:
Can we use a cypress `custom command` to automate this stubbing method?

This should work `cypress/support/commands.js`: 

```
Cypress.Commands.add('stubRequest', (fixtures) => {
  cy.route2('POST', Cypress.env('NEXT_PUBLIC_GRAPHQL_URI'), (req) => {
    const operationName = JSON.parse(req.body).operationName
    if (operationName in fixtures) {
      req.reply((res) => res.send({ fixture: fixtures[operationName] }))
    }
  })
})
```

and into the `beforeEach`: 

```
 beforeEach(() => {
    cy.stubRequest({
      GetTaxonomies: 'taxonomiesList.json',
      GetProducts: 'productList.json'
    })

    cy.visit('/')
  })
```

Reply:
Great idea, looks much cleaner!